src/sass/main.scss

```diff
@@ -1,5 +1,120 @@
 // scss goes here
 
 html {
-  font-family: helvetica;
+  font-family: sans-serif;
+}
+
+// used for theme skins with a default to gray
+@mixin color_cascade($color: #999) {
+  .wand-trigger {
+    border: 2px solid $color;
+    color: $color;
+  }
+  .wand-history-node {
+    border-color: $color;
+    &:before {
+      background-color: white;
+      border: 0.2em solid $color;
+    }
+  }
+  .wand-node-container {
+    border-color: $color;
+    &:before {
+      background-color: $color;
+      border: 0.2em solid $color;
+    }
+  }
+  .wand-history-title {
+    color: $color;
+  }
+}
+
+.wand {
```

Comment:
defer to @phiden & you, but it seems like `wand` has a reasonably high probability of collision. can/should we change this to Wand or WAND or WaNd or something like that?